releases/8.0.0.md

```diff
@@ -139,7 +140,8 @@ JavaScript:
 - `WorkflowHistoryView` moved from `arches.app.views.workflow_history` to `arches.app.views.api`.
 - The format of JavaScript files has changed. Please refer to [Upgrading an Arches project](#upgrading-an-arches-project).
 - The format of how some images are loaded in template files has changed. Please refer to [Upgrading an Arches project](#upgrading-an-arches-project).
-- Update any imports from `@primevue/themes` to `@primeuix/themes`.
```

Comment:
threw in a small change here to move update instructions inside `Upgrading an Arches project` section